test/OgmiosDatumCache.purs

```diff
@@ -27,31 +23,14 @@ import Test.Spec.Assertions (shouldEqual)
 
 suite :: TestPlanM (Aff Unit) Unit
 suite = group "Ogmios Datum Cache tests" $ do
+  -- TODO Move
```

Comment:
Is TODO resolved?

Reply:
I can't remember my intention, probably to move it into Test.Ctl.QueryM. However, we no longer need ODC so I'll be removing this code